packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -773,4 +774,34 @@ export class DashboardModel {
         });
         return this.getById(dashboardUuid);
     }
+
+    async getOrphanedCharts(
+        dashboardUuid: string,
+    ): Promise<Pick<SavedChart, 'uuid'>[]> {
+        const getLastVersionIdQuery = this.database(DashboardsTableName)
+            .leftJoin(
+                DashboardVersionsTableName,
+                `${DashboardsTableName}.dashboard_id`,
+                `${DashboardVersionsTableName}.dashboard_id`,
+            )
+            .select([`${DashboardVersionsTableName}.dashboard_version_id`])
+            .where(`${DashboardsTableName}.dashboard_uuid`, dashboardUuid)
+            .orderBy(`${DashboardVersionsTableName}.created_at`, 'desc')
+            .limit(1);
+
+        const getChartsInTilesQuery = this.database(DashboardTileChartTableName)
```

Comment:
missing await ?

Reply:
No, I just want to prepare/generate these subqueries separately for better readability. 
I only await the last query in this function which includes all these subqueries.